sentry_sdk/integrations/aws_lambda.py

```diff
@@ -116,6 +116,13 @@ def sentry_handler(event, context, *args, **kwargs):
                     )
                     hub.capture_event(event, hint=hint)
                     reraise(*exc_info)
+                finally:
+                    if (
+                        integration.timeout_warning
+                        and configured_time > TIMEOUT_WARNING_BUFFER
+                    ):
```

Comment:
I see that `timeout_thread` is only conditionally defined. That's why you need to repeat this condition here.

It would be clearer to always define `timeout_thread`, initially setting it to `None`.
That way, here it would become:

```py
if timeout_thread is not None:
    timeout_thread.stop()
```

`.stop()` sets a "stop event" to true, unblocking and terminating the thread if still running, and doing nothing otherwise.

Reply:
Made this change in latest commit.